README.md

```diff
@@ -36,6 +38,10 @@ $ npm run build
 $ npm run check-links
 ```
 
+### Cloud IDE
+
+This repository has been setup to use a [Gitpod.io](https://gitpod.io) remote development environment. The for example, visiting [https://gitpod.io/#https://github.com/notaryproject/notaryproject.dev](https://gitpod.io/#https://github.com/notaryproject/notaryproject.dev) launches an in browser editor and preview environment for the [NotaryProject.dev](https://notaryproject.dev) site.
+
```

Comment:
I'd move this to the top (as the first second-level heading), and make it more terse -- maybe something like this:

> ## Cloud build
>
> Visit <https://gitpod.io/#https://github.com/notaryproject/notaryproject.dev> to launch a [Gitpod.io](...) IDE that will allow you to build, preview and make changes to this repo.

Then change the current second-level heading to

> ## Local build

and leave everything else the same.

Reply:
FYI: you'll need to rebase given a small commit I made earlier.